docs/en/index.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Silverstripe elemental
+title: Elemental
 ---
 
-# Silverstripe elemental
+# Elemental
```

Comment:
We should probably say one of "Elemental blocks", "Elemental content" or "Elemental content blocks" so it's a little bit more descriptive of what functionality is provided. Since this sits in the "Optional features" section it should probably be descriptive of a feature.

Reply:
Updated